policy/simple_recipe.js

```diff
@@ -0,0 +1,32 @@
+export const SimpleRecipe = {
+  $meta: {
+    description: 'A very simple recipe',
+  },
+  $stores: {
+    public_texts: {
+      $type: '[Text]',
+      $tags: ['public'],
+      $value: [],
+    },
+    ignored_data: {
+      $type: '[Text]',
+      // $tags: ['public'],
```

Comment:
need?

Reply:
I need J to review this. Originally, J checked in an public tag on that, but I wanted to test the case where it's a store that shouldn't be egressed, so I commented it out. Maybe I can add a non_ignored but not public store instead.